e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -0,0 +1,711 @@
+
+        const card = deserializeCardFromUrl(hash);
+        expect(card.name).to.deep.equal(TARGET_QUESTION.name);
+        expect(card.display).to.deep.equal(TARGET_QUESTION.display);
+        expect(card.dataset_query.query).to.deep.equal({
+          ...TARGET_QUESTION.query,
+          filter: QUERY_FILTER_CREATED_AT,
+        });
+      });
+    });
+
+    it("allows setting saved question with multiple parameters as custom destination", () => {
+      cy.createQuestion(TARGET_QUESTION);
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      addSavedQuestionDestination();
+      addSavedQuestionCreatedAtParameter();
+      addSavedQuestionQuantityParameter();
+      cy.get("aside").button("Done").click();
+
+      saveDashboard();
+
+      clickLineChartPoint();
+      cy.findByTestId("qb-filters-panel").should(
+        "contain.text",
+        "Created At is August 1–31, 2022",
+      );
+      cy.findByTestId("qb-filters-panel").should(
+        "contain.text",
+        "Quantity is equal to 79",
+      );
+      cy.location().should(({ hash, pathname }) => {
+        expect(pathname).to.equal("/question");
+        const card = deserializeCardFromUrl(hash);
+        expect(card.name).to.deep.equal(TARGET_QUESTION.name);
+        expect(card.display).to.deep.equal(TARGET_QUESTION.display);
+        expect(card.dataset_query.query).to.deep.equal({
+          ...TARGET_QUESTION.query,
+          filter: ["and", QUERY_FILTER_CREATED_AT, QUERY_FILTER_QUANTITY],
+        });
+      });
+    });
+
+    it("does not allow setting saved question as custom destination if user has no permissions to it", () => {
+      cy.createCollection({ name: RESTRICTED_COLLECTION_NAME }).then(
+        ({ body: restrictedCollection }) => {
+          cy.updateCollectionGraph({
+            [USER_GROUPS.COLLECTION_GROUP]: {
+              [restrictedCollection.id]: "none",
+            },
+          });
+
+          cy.createQuestion({
+            ...TARGET_QUESTION,
+            collection_id: restrictedCollection.id,
+          });
+        },
+      );
+
+      cy.signOut();
+      cy.signInAsNormalUser();
+
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      cy.get("aside").findByText("Go to a custom destination").click();
+      cy.get("aside").findByText("Saved question").click();
+
+      modal().findByText(RESTRICTED_COLLECTION_NAME).should("not.exist");
+      modal().findByText(TARGET_QUESTION.name).should("not.exist");
+    });
+
+    it("allows setting URL as custom destination and changing it back to default click behavior", () => {
```

Comment:
I wonder if it's deliberate that this is the only test that tests removing the click behavior. Why not testing removing the click behavior for dashboards and saved questions too?

Reply:
I think that testing the "Open drill-through menu" option needs only 1 test, and since it's the default and requires non-default being selected beforehand, it felt reasonable to just add it in one of the existing tests.

I picked this one only because it was the first one that I implemented.

Please let me know if you think this should be added to more (all?) tests.

Reply:
I mean I don't mind that we have to test removing click behavior in all dimensions, but it's good to have that covered. What about adding 2 more tests (or new assertions in existing tests)? One in dashboards and one for questions, so we don't need to add a ton, but still have them covered.

Reply:
> One in dashboards and one for questions, so we don't need to add a ton, but still have them covered.

Only "custom destination > URL" and "updating dashboard filters" do not require navigating back and forth.
I'm kind of hesitant to add it elsewhere because of that, but here it is: 9374f6a3d9e17da8a0668baf76d3426d773c5ece

Reply:
I see what you mean. I haven't accounted for having to navigate back at first.